lib/graphql/schema/resolver.rb

```diff
@@ -22,6 +22,7 @@ class Resolver
       include Schema::Member::GraphQLTypeNames
       # Really we only need description & comment from here, but:
       extend Schema::Member::BaseDSLMethods
+      extend Member::BaseDSLMethods::ConfigurationExtension
```

Comment:
There's a side-effect from this change where resolvers that inherit from other resolvers must now _explicitly_ declare a typename, otherwise they inherit the parent's typename and become a duplicate definition name.

Was this intentional? It's a bit counter-intuitive to the default precedent of elements following their class names.

Reply:
FWIW, @swalkinshaw and I talked about the new behavior and agreed that it isn’t all bad. A resolver is effectively a representation of a Type, and type inheritance is generally discouraged in favor of sharing behavior through mixins. That’s internal precedent though. The former behavior is probably more consistent with the general library.